engine/runtime-instrument-common/src/main/java/org/enso/interpreter/service/ExecutionService.java

```diff
@@ -283,12 +285,17 @@ public CompletionStage<Object> execute(
    * @return a function call with the updated arguments
    */
   private FunctionCallInstrumentationNode.FunctionCall substituteMissingArguments(
-      FunctionCallInstrumentationNode.FunctionCall functionCall) {
+      FunctionCallInstrumentationNode.FunctionCall functionCall, RootNode where) {
     var arguments = functionCall.getArguments().clone();
     var argumentInfos = functionCall.getFunction().getSchema().getArgumentInfos();
+    var errorBuiltins = context.getBuiltins().error();
     for (var i = 0; i < arguments.length; i++) {
       if (arguments[i] == null && !argumentInfos[i].hasDefaultValue()) {
-        arguments[i] = context.getBuiltins().nothing();
+        arguments[i] =
+            DataflowError.withDefaultTrace(
```

Comment:
- I was dreaming of using existing [DataflowError.UNINITIALIZED](https://github.com/enso-org/enso/blob/826486f57909308268f422f28b2310ba193c37c1/engine/runtime/src/main/java/org/enso/interpreter/runtime/error/DataflowError.java#L41)
- but dedicated `makeMissingArgument` with additional info
- is even better